nixos/modules/system/boot/luksroot.nix

```
@@ -294,13 +294,11 @@ let
 
         new_response="$(ykchalresp -${toString yubikey.slot} -x $new_challenge 2>/dev/null)"
 
-        if [ ! -z "$k_user" ]; then
-            new_k_luks="$(echo -n $k_user | pbkdf2-sha512 ${toString yubikey.keyLength} $new_iterations $new_response | rbtohex)"
+        if [ -n "$k_user" ]; then
+            echo -n "$k_user"
```

This might change the behavoir.
```suggestion
            echo -n $k_user
```
would keep the old behavoir.